### DISCUSSION TITLE
SoftFever does not open on Apple Mac mini M1

### DISCUSSION THREAD

**Discussion by Ben240519** [OrcaSlicer/OrcaSlicer#438] [Q&A]
Hello everyone, since today I have a problem: I can no longer open the SoftFever slicer.

The following message appears: "BambuStudio-SoftFever" is damaging your computer. It's a good idea to move the object to the trash.
![Bildschirm­foto 2023-03-07 um 15 47 52](https://user-images.githubusercontent.com/127222722/223460069-56a01f2b-c38f-4e8f-aabb-b488be921585.png)

I noted that in the article.

https://www.howtogeek.com/803598/app-is-damaged-and-cant-be-opened/

Who knows the problem and has a solution?

Mac mini M1 with macOS Ventura version 13.2.1
BambooSlicer SoftFever V1.4.5

Thanks very much!

**Comment by Switch-Witch** [comment]
Same for me (Monterey) ... 
I am so sad... Loved the app!

**Comment by shafnitz** [comment]
Same here. This is new behavior as of today. xattr command does not resolve.

Edit: Original Bambu Studio is still working.
<img width="361" alt="Screenshot 2023-03-07 at 3 55 22 PM" src="https://user-images.githubusercontent.com/61215807/223573023-13aeb90a-5244-4ab9-9721-473893ca80ac.png">

**Comment by shafnitz** [comment]
P.S. There is a issue and discussion here: https://github.com/SoftFever/BambuStudio-SoftFever/issues/427
